libraries/src/Application/WebApplication.php

```diff
@@ -1098,6 +1098,11 @@ public function flushAssets()
 	protected function header($string, $replace = true, $code = null)
 	{
 		$string = str_replace(chr(0), '', $string);
+
+		if ($code === null)
+		{
+			$code = 0;
+		}
```

Comment:
```suggestion
		}

```
For code style it needs an empty line after an if block.

Reply:
No idea why the PHPCS check in drone did not complain about it.